helm_deploy/django-app/templates/ingress.yaml

```diff
@@ -26,13 +26,11 @@ spec:
   {{- end }}
 {{- end }}
   rules:
-  {{- range .Values.ingress.hosts }}
-    - host: {{ . }}
+    - host: {{ .Values.deploy.hosts }}
```

Comment:
Minor thing, but if this has been changed to only expect or accept a single value it'd be best to rename this value to `host`.

Reply:
changed in 6a9c5ec.